datadog_checks_dev/datadog_checks/dev/tooling/utils.py

```diff
@@ -62,14 +62,50 @@ def string_to_toml_type(s):
     return s
 
 
+def complete_set_root(args):
+    """Set the root directory within the context of a cli completion operation."""
+
+    # if we have already set to something other than empty string break
+    if get_root() != '':
```

Comment:
Suggestion that would remove the need for this comment IMO:

```suggestion
    existing_root = get_root()
    if existing_root:
        return
```